src/lib/nvme_pt_ioctl.c

```diff
@@ -328,6 +328,8 @@ void opal_deinit_pt(struct sed_device *dev)
 	}
 
 	opal_parser_deinit();
+
+	dev->fd = 0;
 }
 
 int opal_init_pt(struct sed_device *dev, const char *device_path)
```

Comment:
I think opal_deinit_pt(dev) should be called in every error condition before returning error code -- otherwise there will memory leak in case of error

Reply:
This can be done in a different patch

Reply:
Sure. I would like to make sure that we won't miss it.

Reply:
I will submit another patch for this...